request.js

```diff
@@ -1411,6 +1405,19 @@ Request.prototype.destroy = function () {
     self.response.destroy()
   }
 }
+Request.prototype.enableUnixSocket = function () {
```

Comment:
Also not sure if that needs to be defined on the prototype as a public method, probably a function that accepts an `uri` object will be just fine.